README.md

```diff
@@ -1,5 +1,5 @@
 ![System.IO.Abstractions](https://socialify.git.ci/TestableIO/System.IO.Abstractions/image?description=1&font=Source%20Code%20Pro&forks=1&issues=1&pattern=Charlie%20Brown&pulls=1&stargazers=1&theme=Dark)
-[![NuGet](https://img.shields.io/nuget/v/System.IO.Abstractions.svg)](https://www.nuget.org/packages/System.IO.Abstractions)
+[![NuGet](https://img.shields.io/nuget/v/TestableIO.System.IO.Abstractions.svg)](https://www.nuget.org/packages/TestableIO.System.IO.Abstractions)
 ![Continuous Integration](https://github.com/TestableIO/System.IO.Abstractions/workflows/Continuous%20Integration/badge.svg)
 [![Codacy Badge](https://app.codacy.com/project/badge/Grade/54479b054d194adfb4ff476ef0182fe0)](https://www.codacy.com/gh/TestableIO/System.IO.Abstractions/dashboard?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=TestableIO/System.IO.Abstractions&amp;utm_campaign=Badge_Grade)
```

Comment:
The codacy badge seems incorrect on the `refactor/packaging` branch:
![image](https://user-images.githubusercontent.com/3438234/204299335-f1897336-5445-4379-ae71-c5c6c37f8f88.png)

Reply:
Seems this was a temporary glitch, it works now:

<img width="776" alt="image" src="https://user-images.githubusercontent.com/1540469/204304610-702e0383-c6df-419d-b0d5-395b0d97d7bf.png">

Reply:
Yes, you are right.
I just saw it when reading the updated README.md...